docs/guides/quickstart-dfp.md

```diff
@@ -0,0 +1,258 @@
+
+As the first step, you will configure a BloxOne Host on AWS with DFP service.
+
+You will use the following module to create it:
+- [bloxone_infra_host_aws](https://github.com/infobloxopen/terraform-provider-bloxone/tree/master/modules/bloxone_infra_host_aws)
+
+The module requires the [AWS terraform provider](https://registry.terraform.io/providers/hashicorp/aws/latest) to be configured.
+To configure the AWS provider, add the following code to your main.tf:
+
+````terraform
+provider "aws" {
+  region     = "us-west-2"
+  access_key = "my-access-key"
+  secret_key = "my-secret-key"
+}
+```` 
+
+!> Warning: Hard-coded credentials are not recommended in any configuration file. It is recommended to use the AWS credentials file or environment variables.
+
+You can also use the following environment variables to configure the provider:
+`AWS_REGION`, `AWS_ACCESS_KEY_ID`, and `AWS_SECRET_ACCESS_KEY`.
+
+To create an EC2 instance with DHCP service, you will need to have the following information:
+- key_name: The name of the key pair to use for the instance
+- subnet_id: The ID of the subnet to launch the instance into
+- vpc_security_group_ids: A list of security group IDs to associate with the instance
+
+Add the following code to your main.tf to create an EC2 instance with DHCP service:
+
+````terraform
+
+// Create a BloxOne Host on AWS with DHCP service
+module "bloxone_infra_host_aws" {
+  source = "github.com/infobloxopen/terraform-provider-bloxone//modules/bloxone_infra_host_aws"
+  
+  key_name               = "my-key"
+  subnet_id              = "subnet-id"
+  vpc_security_group_ids = ["vpc-security-group-id"]
+
+  services = {
+    dfp = "start"
+  }
+}
+````
+
+You will need the pool ID of the AWS host to create the Infra Service block for DFP. 
+
+To create the Infra service block , we use the following resource :
+- [bloxone_infra_service](https://registry.terraform.io/providers/infobloxopen/bloxone/latest/docs/resources/infra_service)
+
+Add the following code to your main.tf:
+
+````terraform
+resource "bloxone_infra_service" "example" {
+  name         = "example_dfp_service"
+  pool_id      = data.bloxone_infra_hosts.dfp_host.results.0.pool_id
+  service_type = "dfp"
+  desired_state = "start"
+}
+````
+
+
+Next , we create the DFP Service block and an Internal Domain List using the following resources:
+- [bloxone_td_internal_domain_list](https://registry.terraform.io/providers/infobloxopen/bloxone/latest/docs/resources/td_internal_domain_list)
+- [bloxone_dfp_service](https://registry.terraform.io/providers/infobloxopen/bloxone/latest/docs/resources/dfp_service)
+
+We use the service ID of the Infra Service block to create the DFP Service.
+
+Add the following code to your main.tf:
+
+````terraform
+resource "bloxone_td_internal_domain_list" "example_list" {
+  name = "example_internal_domain_list"
+  internal_domains = ["example.domain.com"]
+}
+
+# Create the DFP Service
+resource "bloxone_dfp_service" "example" {
```

Comment:
Match the name 'bloxone_dfp_service' in all cases. In previous step, Infra service was created with name 'example_dfp_service' and here, we mentioned resource with different name.

Reply:
Fixed